platform/ios/Integration Tests/MGLMapViewIntegrationTest.m

```diff
@@ -78,6 +78,27 @@ - (void)waitForMapViewToBeRenderedWithTimeout:(NSTimeInterval)timeout {
     [self waitForExpectations:@[self.renderFinishedExpectation] timeout:timeout];
 }
 
+- (void)waitForExpectations:(NSArray<XCTestExpectation *> *)expectations timeout:(NSTimeInterval)seconds {
+
+    NSTimer *timer;
+
+    if (@available(iOS 10.0, *)) {
+        // We're good.
+    }
+    else if (self.mapView) {
+        // Before iOS 10 it seems that the display link is not called during the
+        // waitForExpectations below
+        timer = [NSTimer scheduledTimerWithTimeInterval:1.0/30.0
+                                                 target:self.mapView
+                                               selector:NSSelectorFromString(@"updateFromDisplayLink")
```

Comment:
This is clever, but a bit scary. Would a selector literal make this any more future-proof?

Reply:
Unfortunately (and rightly so) that method isn't exposed by MGLMapView. One thought I had was to add the declaration via a category. That would look nicer here, but might be confusing.

At least using `NSSelectorFromString` screams "don't do this"!

Reply:
Couldn't do it. Switched to `@selector` and the category declaration 😄